scalding-parquet/src/main/scala/com/twitter/scalding/parquet/tuple/macros/Macros.scala

```diff
@@ -0,0 +1,50 @@
+package com.twitter.scalding.parquet.tuple.macros
+
+import com.twitter.scalding.parquet.tuple.macros.impl.{ ParquetSchemaProvider, ParquetTupleConverterProvider, WriteSupportProvider }
+import com.twitter.scalding.parquet.tuple.scheme.ParquetTupleConverter
+import parquet.io.api.RecordConsumer
+import parquet.schema.MessageType
+
+import scala.language.experimental.macros
+
+object Macros {
```

Comment:
hi @julienledem , I will commit a PR in parquet project once it comes to release then I will commit a PR in scalding to delete these macros. Right now could we let these macros stay in Scalding?